other/postgres-cleanup.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+psql -d $POSTGRES_DB -U $POSTGRES_USER -c "DELETE FROM bgp_updates WHERE timestamp < NOW() - interval '1 hours' AND handled=true AND hijack_key=ARRAY[]::text[];"
```

Comment:
please make this configurable with an env variable (preferrably in number of days). Also remove unhandled updates too, no need to keep them in